TrainingOOP-PHP001/studentManagement.php

```diff
@@ -0,0 +1,92 @@
+<?php
+namespace TrainingOOP-PHP000PHP0001;
+//Array to store student list
+$listStudent = array();
+
+// function : addStudent
+function addStudent($name, $point) {
+    global $listStudent;
+    $Student = array("name" => $name, "point" => $point);
```

Comment:
Nên dùng `[]`.
Tên biến không nên viết hoa.
```suggestion
    $student = ["name" => $name, "point" => $point];
```